src/createActions.js

```diff
@@ -53,3 +53,11 @@ function fromIdentityActions(identityActions) {
     , {})
   );
 }
+
+function camelCase(word) {
+  return word.split(/[^a-zA-Z0-9]+/).map((part, index) => (
+    index === 0 ?
+      part.toLowerCase() :
+      part[0].toUpperCase() + part.slice(1).toLowerCase()
+  )).join('');
+}
```

Comment:
Could you please write provide some tests as well? Just to make sure it works correctly and we don't break it in the future ;)